### PR TITLE
Fix databricks sdk references for publishing

### PR DESCRIPTION
**Pull request by zavoraad**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-databricks-api==0.9.0
\ No newline at end of file
+databricks-sdk
```
Comment (aamend) [inline requirements.txt]: Do we want to restrict this job to a specific version?

File: setup.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
     include_package_data=True,
     install_requires=[
         'databricks-api==0.9.0',
+        'databricks-sdk',
```
Comment (aamend) [inline setup.py]: Same as above, restrict to a specific version we know works so that we avoid same pain later

**Review by aamend — CHANGES_REQUESTED** [review]
Can you just specify databricks-sdk version to `databricks-sdk==0.38.0`

File: requirements.txt (hunk 1 above)
Comment (zavoraad) [reply requirements.txt]: Yes, done

File: setup.py (hunk 2 above)
Comment (zavoraad) [reply setup.py]: Yes, done

**Review by aamend — APPROVED** [review]

**Pull request merged**